src/createAuthUser.js

```diff
@@ -158,13 +161,15 @@ const createAuthUser = ({
     phoneNumber = deserializedUser.phoneNumber
     displayName = deserializedUser.displayName
     photoURL = deserializedUser.photoURL
+    tenantId = deserializedUser.tenant
```

Comment:
I believe this will be `deserializedUser.tenantId`. Please add a test to confirm.